lib/event.c

```diff
@@ -570,6 +571,9 @@ struct event_loop *event_master_create(const char *name)
 		rv->fd_limit = (int)limit.rlim_cur;
 	}
 
+	if (limit_fds)
```

Comment:
it's a problem for any process to try to use an unreasonable number, isn't it? let's make a reasonable default upper limit, and then let someone who really knows what they're doing ask for a bigger number explicitly. like, limit to MIN(fd_setsize, FRRLIMIT), and then exceed that limit only if specifically asked to.
what'd that upper-limit be; what'd make sense for bgp, especially? 10K? 64K?

Reply:
fixed